samples/unit-tests/axis/label-ellipsis/demo.js

```diff
@@ -69,7 +69,7 @@ QUnit.test('#5034: No ellipsis for multiline labels where there is room', functi
 
         chart: {
             type: "bar",
-            width: 500
+            width: 450
```

Comment:
How can this be necessary?

Reply:
This is not related to the TS conversion, but after pulling latest changes from master this was needed as this test fails on Chrome/Linux. Thus, I could push without fixing the test as well.